core/src/main/java/org/zenoss/zep/zing/impl/ZingEventProcessorImpl.java

```diff
@@ -167,9 +173,18 @@ private void processEventSummary(EventSummary summary) {
         }
 
         ZingEvent zingEvent = builder.build();
+
+        PubsubMessage pubsubMessage = this.publisher.getPubSubMessage(zingEvent);
+        int pubsubMessageSize = pubsubMessage.getData().size();
```

Comment:
Lines 177 and 178 should be moved inside the `if (zingEvent.isValid()) {}` block